model/articles.models.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../db/connection')
+
+
+exports.fetchArticlebyId = (id) => {
+    return db.query('SELECT * FROM articles WHERE article_id = $1', [id])
+    .then((response) => {
+        if (response.rows.length === 0) {
+           return Promise.reject({ status: 404, msg: 'Not Found'})
+        } else {
+        return response.rows[0]
+        }
+    })
+}
+
+
+exports.updateArticleById = (id, updateVoteBy) => {
+    return db.query(
+    `UPDATE articles
+    SET votes = $1
```

Comment:
Please change this, currently this will just overwrite the values. So if you're testing an articles whos vote is at 0, this will pass. However if the article has any other number of votes, it will fail. If its starts at 3 and you add 20, this will return that the votes === 20 not 23.